Added detroitWebDev

Imran3917 opened the pull request:
Project was added on 08/31/2021

MaxJewell commented:
Could we have a link to a twitter or website with the Policy ID's on them.

Thanks.

Imran3917 commented:
Hi, This is my twitter profile link: https://twitter.com/detroitWebDv ; I also added one new NFT and added the policy ID on GitHub.  The codes are below in case you need it. Please let me know if you need any more information.

{
    "project": "detroitWebDev",
    "policies": [
        "6fea7b2f0bf99451c52d061c1619f683ebbb0f7a5aaa7ed895315127",
        "493de73305c0cdae06727286f40ec67bdf6bf8716bb75ab0a620aad2",
        "2d1e2b2e66782a32ae3ad638120bf91a1be9f44d8a0cee13651defee"
    ]
}

Imran Ahmed ***@***.***) | Twitter<https://twitter.com/detroitWebDv>
The latest Tweets from Imran Ahmed ***@***.***): &quot;https://t.co/2Sjtewixuo&quot;
twitter.com
Thanks,

Imran
________________________________
From: MaxJewell ***@***.***>
Sent: Tuesday, September 21, 2021 6:11 AM
To: Cardano-NFTs/policyIDs ***@***.***>
Cc: Imran3917 ***@***.***>; Author ***@***.***>
Subject: Re: [Cardano-NFTs/policyIDs] Added detroitWebDev (#621)


Could we have a link to a twitter or website with the Policy ID's on them.

Thanks.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Cardano-NFTs/policyIDs/pull/621#issuecomment-923833019>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AKLJ46KPVRXSGJFYIJY7EG3UDBK3LANCNFSM5DE2H6ZA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

Imran3917 commented:
Hi, I still don’t see my project on cnft.io, did make any mistakes? Please advise? [cid:B1AFBB35-5048-4B88-B401-926AE5B41ADE-L0-001]

Sent from my iPhone

On Sep 28, 2021, at 4:48 AM, MaxJewell ***@***.***> wrote:

﻿

Closed #621<https://github.com/Cardano-NFTs/policyIDs/pull/621>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Cardano-NFTs/policyIDs/pull/621#event-5372207105>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AKLJ46IFY4KUYZE2YHQNP73UEF6PNANCNFSM5DE2H6ZA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
